oidc-service-impl/src/main/java/io/mosip/esignet/services/LinkedAuthorizationServiceImpl.java

```diff
@@ -150,11 +158,14 @@ public LinkTransactionResponse linkTransaction(LinkTransactionRequest linkTransa
 
         //Publish message after successfully linking the transaction
         kafkaHelperService.publish(linkedSessionTopicName, linkCodeHash);
+        auditWrapper.logAudit(Action.LINK_TRANSACTION, ActionStatus.SUCCESS, AuditHelper.buildAuditDto(linkTransactionMetadata.getTransactionId(), transaction), null);
         return linkTransactionResponse;
     }
 
     @Override
     public OtpResponse sendOtp(OtpRequest otpRequest) throws IdPException {
+        authorizationHelperService.validateCaptchaToken(otpRequest.getCaptchaToken());
```

Comment:
Pls remove this, we will not have a captcha token during linked transaction.

Reply:
Removed this line.